### PR TITLE
Adding Information about Rajesh

### PR DESCRIPTION
**Pull request by sgunique**
please review my changes...

### CODE REVIEW

**Comment by rajeshg03** [comment]
why did you make this change ?

**Comment by sgunique** [comment]
This just to validate test workflow.

File: _posts/0000-01-01-intro.md [hunk 1]
```diff
@@ -4,3 +4,5 @@ title: "Welcome to our slide deck!"
 ---
 
 Use the right arrow to begin!
+
+Rajesh is validating.
```
Comment (rajeshg03) [inline _posts/0000-01-01-intro.md]: pls delete this text

File: _includes/head.html [hunk 2]
```diff
@@ -19,4 +19,5 @@
 	link.type = 'text/css';
 	link.href = window.location.search.match( /print-pdf/gi ) ? 'css/print/pdf.css' : 'node_modules/reveal.js/css/print/paper.css';
 	document.getElementsByTagName( 'head' )[0].appendChild( link );
-</script>
\ No newline at end of file
+  console.log("Test");
```
Comment (rajeshg03) [inline _includes/head.html:22]: delete this

Comment (sgunique) [reply _includes/head.html:22]: sure I will delete. thanks for finding bug

File: _posts/0000-01-01-intro.md (hunk 1 above)
Comment (sgunique) [reply _posts/0000-01-01-intro.md]: No.. we this due to some reason.

**Review by rajeshg03 — APPROVED** [review]

**Pull request merged**